.github/workflows/ci.yml

```diff
@@ -0,0 +1,156 @@
+# CI WorkFlow for build, test and release of Plugin Packages
+name: CI for Plugin Packages
+on:
+  push:
+    branches:
```

Comment:
we can remove `main` from here 
just trigger on tag

Reply:
yes.. done. Please review